tools/tsdb-print-chunk/main.go

```diff
@@ -52,9 +53,14 @@ func main() {
 		fmt.Println("Chunk ref:", ref, "samples:", ch.NumSamples(), "bytes:", len(ch.Bytes()))
 
 		it := ch.Iterator(nil)
-		for it.Err() == nil && it.Next() {
-			ts, val := it.At()
-			fmt.Printf("%g\t%d (%s)\n", val, ts, timestamp.Time(ts).UTC().Format(time.RFC3339Nano))
+		for valType := it.Next(); it.Err() == nil && valType != chunkenc.ValNone; valType = it.Next() {
```

Comment:
[nit] By contract, `Next()` must return `ValNone` on error.

```suggestion
		for valType := it.Next(); valType != chunkenc.ValNone; valType = it.Next() {
```

Reply:
fixed